RecoLocalCalo/HGCalRecProducers/interface/HGCalLayerTiles.h

```diff
@@ -90,6 +90,9 @@ class HGCalLayerTilesT {
   }
 
   std::array<int, 4> searchBoxEtaPhi(float etaMin, float etaMax, float phiMin, float phiMax) const {
+    if (etaMax - etaMin < 0) {
```

Comment:
(just to check): no swap is needed here?

Reply:
Ciao @perrotta , thanks for asking. In this specific case, the swap is not needed because this data structure internally supports both positive and negative eta values. The check that the _extension_ of the interval in eta makes sense is the only one needed. The rest is taken care of internally by the bin assignment.